Lesson_5/task1.py

```diff
@@ -0,0 +1,27 @@
+# Task 1 - The greatest number
+# Write a Python program to get the largest number from a list of random numbers
+# with the length of 10
+# Constraints: use only while loop and random module to generate numbers
+
+import random
+
+LIST_LENGTH = 10
+new_list = []
+
+# An alternative way to create a list of random numbers with the length of 10
+# new_list = random.sample(range(1, 100), k=LIST_LENGTH)
+
+while len(new_list) < LIST_LENGTH:
+    new_list.append(random.randint(0, 100))
+else:
+    largest_num = new_list[0]
```

Comment:
This block should not have been included in `else`. `Else` makes sense only when we use `break`, otherwise else is processed always (just like the code outside the block)

Reply:
+